### DISCUSSION TITLE
Support sorting by pushed to default branch

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#44021] [Mobile] [answered]
My organization has dependabot enabled, so sorting by `recently pushed` just shows a random list of repositories that dependabot is monitoring.

![Screenshot_20230111-170948.png](https://user-images.githubusercontent.com/2119212/211928874-bdb65413-fb61-41b4-b451-4c2329cd0e6f.png)

The ideal sort would be pushed by me (or certain other users, or ignoring bots), but, at least, sorting by pushes to the default branch would yield something less useless.

I know I can sort by newest, but at times I'll actually want to see the repositories I've refreshed as opposed to when I forked them which could easily be years ago.

**Comment by stevepopovich** [comment, marked as answer]
Hi @jsoref I passed this on to the team for consideration
